tensorboard/plugins/scalar/scalars_plugin.py

```diff
@@ -69,6 +70,7 @@ def get_plugin_apps(self):
         return {
             "/scalars": self.scalars_route,
             "/tags": self.tags_route,
+            "/scalarsmulti": self.scalars_multirun_route,
```

Comment:
I like your choice of the name `scalars_multirun` for the handler
method. Could we maybe call the route `/scalars_multirun`, too? It seems
clearer than `scalarsmulti`, since it’s only “multi” along one of the
two extra axes (and of course the normal `/scalars` is already “multi”
along the time axis).

(There’s inconsistent precedent for both `camelCase` and `snake_case` in
route names (`individualImage`, `plugins_listing`), but most things use
snake case, and we don’t tend to use `lowersmooshcase`.)